docs/src/pages/de/installation.md

```diff
@@ -0,0 +1,176 @@
+```
+
+Nachdem `create-astro` Ihr Projekt vorbereitet hat, denken Sie daran, Ihre Projektabhängigkeiten mit npm oder dem Paketmanager Ihrer Wahl zu installieren. In diesem Beispiel verwenden wir npm:
+
+```bash
+npm install
+```
+
+Sie können Ihr Astro-Projekt nun [Starten](#start-astro). Nachdem Sie Ihr Astro-Projekt fertiggestellt haben, können Sie Ihr Projekt [Bauen](#build-astro). Astro wird dann die statischen Seiten generieren und für Sie bereitstellen, welche Sie anschließend bei Ihrem bevorzugten Hosting-Anbieter [Ausrollen](/guides/deploy) können.
+
+
+## Manuelle Installation
+
+Sie können Astro auch ohne die Hilfe des `create-astro` Assistenten einrichten. Nachfolgend sind die wenigen zusätzlichen Schritte aufgeführt, die erforderlich sind, um Astro zum Laufen zu bringen. 
+
+### Ihr Projekt aufsetzen
+
+```bash
+# Verzeichnis erzeugen und hinein wechseln
+mkdir my-astro-project
+cd my-astro-project
+```
+
+Erstellen Sie ein leeres Verzeichnis mit dem Namen Ihres Projekts und navigieren Sie dann in dieses:
+
+### `package.json` erzeugen
+
+```bash
+# Dieser Befehl erstellt eine grundlegende package.json Datei für Sie 
+npm init --yes
+```
+
+Astro wurde entwickelt, um mit dem gesamten npm-Paket-Ökosystem zu arbeiten. Dies wird durch ein Projektmanifest im Stammverzeichnis Ihres Projekts verwaltet, das als `package.json` bekannt ist. Wenn Sie mit der Datei `package.json` nicht vertraut sind, empfehlen wir Ihnen dringend, sich ein wenig einzulesen [npm-Dokumentation] (https://docs.npmjs.com/creating-a-package-json-file). 
+
+### Astro installieren
+
+Wenn Sie die obigen Anweisungen ausgeführt haben, sollten Sie ein Verzeichnis mit einer einzelnen `package.json` Datei darin haben. Sie können Astro jetzt in Ihrem Projekt einrichten. 
+
+```bash
+npm install astro
+```
+Sie können jetzt den Platzhalter-Abschnitt "scripts" Ihrer `package.json` Datei, welche durch `npm init` für Sie erstellt wurde, durch Folgendes ersetzen: 
+
+```diff
+  "scripts": {
+-    "test": "echo \"Error: no test specified\" && exit 1"
++    "dev": "astro dev",
++    "build": "astro build",
++    "preview": "astro preview"
+  },
+}
+```
+Der Befehl [`dev`](#start-astro) startet den Astro Entwicklungs Server auf `http://localhost:3000`. Sobald Ihr Projekt fertig ist, generiert der Befehl [`build`](#build-astro) Ihr fertiges Projekt in das Verzeichnis `dist/` . [Lesen Sie mehr über das Ausrollen im Deployment Leitfaden.](/guides/deploy) 
+
+### Erstellen Sie Ihre erste Seite
+
+
+Öffnen Sie Ihren bevorzugten Texteditor und erstellen Sie eine neue Datei in Ihrem Projekt: 
+
+1. Erstellen Sie eine neue Datei unter `src/pages/index.astro`
+2. Kopieren Sie den folgenden Code-Schnipsel (einschließlich `---` Bindestriche) und fügen Sie ihn ein. 
+
+```astro
+---
+// JavaScript/TypeScript-Code, der zwischen dem (---) Bindestrichen geschrieben wurde, wird ausschließlich auf dem Server ausgeführt!
+console.log('Sehen Sie mich im Terminal ')
```

Comment:
```suggestion
console.log('Sehe mich im Terminal ')
```

Reply:
I think logging `'Man sieht mich im Terminal.'` sounds better.

Also there is a space at the end.

Reply:
Since it is a log anyway it could better point to a 'read'

console.log('Lies mich im Terminal')